packages/ui/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,77 @@
+import NextLink, { type LinkProps } from 'next/link';
+import type { HTMLProps, Ref, RefObject } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { Icon } from './Icon';
+import { type BaseProps, LinkSize, LinkVariant } from './types';
+
+export function Link({
+    variation = 'primary',
+    className,
+    forwardRef,
+    forceRefresh,
+    size = 'default',
+    icon,
+    iconPlacement = 'left',
+    disabled,
+    children,
+    contentClassName,
+    localeCode,
+    href,
+    ...linkProps
+}: Link.Props) {
+    const Element = forceRefresh ? 'a' : NextLink;
+
+    const isIconOnly = Boolean(Icon) && !children;
```

Comment:
I don't think a Link can be icon-only. That is true for buttons, but links are usually intended to be placed inside text, so an icon-only Link would be really weird to see.